mdpow/tests/test_solv_shell.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import absolute_import
+
+import numpy as np
+
+from . import tempdir as td
+
+import py.path
+
+import pybol
+import pytest
+
+from numpy.testing import assert_almost_equal
+from scipy.stats import variation
+
+from ..analysis.ensemble import Ensemble, EnsembleAnalysis, EnsembleAtomGroup
+
+from ..analysis.solvation import SolvationAnalysis
+
+from pkg_resources import resource_filename
+
+RESOURCES = py.path.local(resource_filename(__name__, 'testing_resources'))
+MANIFEST = RESOURCES.join("manifest.yml")
+
+
+class TestDihedral(object):
+    mean = 181.71428571428572
+    var = 0.9960847721325709
+
+    def setup(self):
+        self.tmpdir = td.TempDir()
+        self.m = pybol.Manifest(str(RESOURCES / 'manifest.yml'))
+        self.m.assemble('example_FEP', self.tmpdir.name)
+        self.ens = Ensemble(dirname=self.tmpdir.name, solvents=['water'])
+        self.solute = self.ens.select_atoms('not resname SOL')
+        self.solvent = self.ens.select_atoms('resname SOL and name OW')
+
+
+    def teardown(self):
+        self.tmpdir.dissolve()
+
+    def test_dataframe(self):
+        solv = SolvationAnalysis(self.solute, self.solvent, [1.2]).run(start=0, stop=4, step=1)
+
+        for d in solv.results['distance']:
+            assert d == 1.2
+        for s in solv.results['solvent']:
+            assert s == 'water'
+        for i in solv.results['interaction'][:12]:
+            assert i == 'Coulomb'
+
+    def test_selection(self):
+        solv = SolvationAnalysis(self.solute, self.solvent, [2, 10]).run(start=0, stop=4, step=1)
+        mean = np.mean(solv.results['quantity'])
+        var = variation(solv.results['quantity'])
```

Comment:
just use `np.std`; `variation` does not give any new information because you already tested mean and you can avoid 
* using something that I had to first look up, which really annoyed me more than it should have: First "Is `variation()` a function that you defined?" — No. "Where does it come from?" It's imported from `scipy.stats` (sidenote: don't do direct imports, keep it in it's namespace, especially if only used once... `stats.variance` or even clearer `scipy.stats.variance` is roughly 1000 times more obvious), finally "What does it do?" mean/std. "Is it actually important for the test?" NO! 
* importing an additional package

Reply:
done